social_protection/gql_mutations.py

```diff
@@ -154,6 +154,8 @@ def _mutate(cls, user, **data):
 
         service = BenefitPlanService(user)
         if SocialProtectionConfig.gql_check_benefit_plan_update:
+            if 'max_beneficiaries' not in data:
+                data['max_beneficiaries'] = None
```

Comment:
@sniedzielski @delcroip Apologies about the delayed code review as I was on leave. I don't think this change complies with the convention used in this project so far where missing fields are treated as "no update" instead of null/None. Also I believe GraphQL convention is to treat an optional field's absence as "no update" rather than "setting to None" because it's a feature explicitly supported: https://github.com/graphql-dotnet/conventions/issues/128 

Wouldn't it be sufficient on the frontend to set max_beneficiaries=null when the form has empty value and just leave the backend as before?

Reply:
@sniedzielski  please have a look